### DISCUSSION TITLE
PPOCRLabel标注闪退

### DISCUSSION THREAD

**Discussion by 555myhair** [PaddlePaddle/PaddleOCR#12709] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：  Anaconda 
- 版本号/Version：Paddle：  PaddleOCR：v2.7.1 问题相关组件/Related components：
- 运行指令/Command Code：Python PPOCRLabel.py --lang ch
- 完整报错/Complete Error Message： ppocr WARNING:When args.layout is false, args.ocr is automatically set to false.

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

PPOCRLabel打开后，打开照片，点击矩形标注之后卡顿直接闪退。点击多点标注之后，没有识别结果，只有检测框位置。

**Comment by 555myhair** [comment]
退出代码为-107374071（0xC0000409）

**Comment by g39088902** [comment]
可以等待 #11236 合并

**Comment by monkeycc** [comment]
我也是这个问题

通过自己摸索

paddlepaddle 要安装2.4版本

2.5版本就会直接闪退

**Comment by SWHL** [comment]
因PPOCRLabel已经搬家了，后续有问题请移步[PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel)提问哈！
此issue先行关闭了。
